src/Contract/Operation/Matchingable.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace loophp\collection\Contract\Operation;
+
+use Doctrine\Common\Collections\Criteria;
+use Doctrine\Common\Collections\Selectable as DoctrineCollectionsSelectable;
+use loophp\collection\Contract\Collection;
+
+/**
+ * @template TKey
+ * @template T
+ *
+ * @template-extends DoctrineCollectionsSelectable<TKey, T>
```

Comment:
```suggestion
 * @extends DoctrineCollectionsSelectable<TKey, T>
```

have you tried it like this? Could this be causing the errors in the baseline?

Reply:
Yep, it doesn't change anything.